src/Angor/Client/Pages/Recover.razor

```diff
@@ -621,11 +554,12 @@
     private bool showRecoveryModal = false;
     private bool showRecoveryReleaseModal = false;
     private bool showEndOfProjectModal = false;
-    private bool showChatModal = false;
+    private bool showMessageModal = false;
     private bool showNsec = false;
     private string nsecValue = "";
     private string npubValue = "";
     private string founderNpub = "";
+    private string currentUserPrivateKeyHex = "";
```

Comment:
Perhaps we should have it in a component?

Reply:
What do you mean exactly? the popup is already a component